source/frontend/package.json

```diff
@@ -138,6 +140,9 @@
     "pretty-quick": "pretty-quick"
   },
   "jest": {
+    "transform": {
```

Comment:
This helps with ts compilation within jest, in my testing gives a 10 - 15% performance improvement.